### DISCUSSION TITLE
Can't backup using github storage option

### DISCUSSION THREAD

**Discussion by OlegPasko** [requarks/wiki#3543] [Error / Bug Report]
**Describe the bug**

When I try to use Github as a storage following the instruction (https://docs.requarks.io/storage/git) I'm getting:
> 'Warning: Identity file /etc/wiki/github not accessible: No such file or directory. git@github.com: Permission denied (publickey). fatal: Could not read from remote repository. Please make sure you have the correct access rights and the repository exists. error: Could not fetch origin'

**To Reproduce**
1. Setup new server based on DigitalOcean marketplace solution (https://docs.requarks.io/install/digitalocean) including ssl setup.
2. Create a new private repository in git@github.com:team/repo.git (tried 2 options – inited with readme and empty repo).
3. Setup Storage in admin panel following with all the steps from https://docs.requarks.io/storage/git. 
4. No sync with mentioned message.

**Expected behavior**
Github synchronised with Wiki data.

**Screenshots**
<img width="268" alt="Screenshot 2020-08-28 at 10 45 09" src="https://user-images.githubusercontent.com/788079/91535449-d7ac3f00-e91b-11ea-83fe-ed2387cea0cc.png">

**Host Info (please complete the following information):**
 - OS: Prefigured DO droplet
 - Wiki.js version: 2.4.107
 - Database engine: PG 11.8

**Comments**
I believe this is high-demand feature and it's likely I missed something. But I repeated the steps pretty careful.

**Comment by OlegPasko** [comment]
My answers:
Looks I had to refer to the key inside docker container.
I changed `path` method to `contain`. But still get another error that it can't be synced.
So I logged into container from the droplet with `docker exec -it wiki /bin/bash`, manually went to `data/repo` and executed `git push -u origin master`. After I pushed it to the github it works.
I think it makes sense to check 2 points:
1. Clarify instruction for docker in the doc.
2. Check what is wrong with the first push.

ps. Looking forward to having 'Publish' option :)

**Comment by MarsWarrior** [comment]
I'm using a path for the pem file. I had to change the protection of that file (read to everyone), to make it readable for the node user that is used in the official requarks image.

After that I had to restart the docker container to get GIT working.

**Comment by jlippold** [comment]
FWIW it was easier for me to paste the contents of the pem file into the UI.

Also, I had to make at least one commit first to master before syncing would work
